config/webpack.js

```diff
@@ -111,21 +125,18 @@ module.exports = (env) => {
       new HtmlWebpackPlugin({
         inject: false,
         filename: 'index.html',
-        template: '!handlebars-loader!src/index.html',
+        template: '!handlebars-loader!src/index.html'
       }),
 
       new HtmlWebpackPlugin({
         inject: false,
         filename: 'return.html',
-        template: '!handlebars-loader!src/return.html',
+        template: '!handlebars-loader!src/return.html'
       }),
 
       new CopyWebpackPlugin([
-        {
-          from: './node_modules/zip-js/WebContent',
-          to: 'static/zipjs',
-          ignore: ['tests/**/*'],
```

Comment:
I switched to specifically copying 2 files, instead of ignoring some but not all the files we don't need.